composer.json

```diff
@@ -10,7 +10,7 @@
 	},
 	"require": {
 		"elasticsearch/elasticsearch": "~1.3",
-		"illuminate/support": "~4.2"
+		"illuminate/support": "~5.0"
```

Comment:
Alright, will do that.
I was originally thinking that there would be a tagged version now, with `4.2` and a new tagged version with `5.0`.